rest/src/main/java/de/starwit/rest/controller/EntityController.java

```diff
@@ -125,13 +126,14 @@ public EntityDto findByIdAndApp(@PathVariable("appId") Long appId, @PathVariable
 
     @ExceptionHandler(value = { EntityNotFoundException.class })
     public ResponseEntity<Object> handleException(EntityNotFoundException ex) {
-        LOG.info("Entity not found. {}", ex.getMessage());
-        return new ResponseEntity<>("Entity not found.", HttpStatus.NOT_FOUND);
+        NotificationDto output = new NotificationDto("error.entitynotfound", "Entity not found.");
```

Comment:
Sind jetzt alle Keys in lowercase?

Reply:
Die waren ursprünglich alle mal in lowercase - ja. Den Grund für die Änderung kann ich jetzt nicht nachvollziehen. Wir können da gern nochmal ein Ticket machen und die translation keys nochmal durchgehen.

Reply:
geändert in error.entity.notfound damit das gleich zu den anderen Schlüsseln ist.